### DISCUSSION TITLE
Zoeken en vinden

### DISCUSSION THREAD

**Discussion by sytskevanhasselt** [Klantinteractie-Servicesysteem/KISS-frontend#32] [General]
In onderdeel E van de userstories worden de volgende bronnen onderscheiden:
- E-01 Kennisartikelen 
  - (dit zijn zowel gestructureerde data (PDC) 
  -  als ongestructureerde data (mediawiki))
- E-02 Eigen Website (ongestructureerde data)
- E-03 Smoelenboek (gestructureerde data)
- E-04 Zaaksysteem (gestructureerde data)
 
In onderdeel D van de userstories staat de eisen aan de werking van het zoeken en de presentatie van het zoekresultaat
- D-02 In meerdere bronnen zoeken
- D-06 Presenteren van de content van de bronnen binnen de KISS interface
 
**We hebben hierover het volgende afgesproken:**

1. Om te kunnen voldoen aan de wens om alle resultaten in een overzicht te presenteren gaat alles via Elastic Search (muv zoeken in zaaksysteem zie punt 3).
2. Bij de gestructureerde bronnen wordt de Elastic index gevuld via de daarvoor beschikbare API’s (bijv. de PDC API).
3. Zaaksysteem (gestructureerde data) gaan we niet via Elastic doen. Oa ivm authenticatie op de zaken. Je gaat ook niet vrij zoeken in zaken, maar alleen o.b.v. bsn of zaaknummer de zaken rond een persoon ophalen. Vrij zoeken naar zaken is geparkeerd.
4. Van de gestructureerde bronnen kunnen we de content binnenhalen naar de interface van KISS.
5. Voor de ongestructureerde bronnen kan dat niet. In dat geval geven we de link naar de bron.

**Comment by rubenvdlinde** [comment]
Te bespreken in het architectuur overleg van 16 juni 12:30 tot 14:30

Wie zijn er nodig
- [ ] Architecten
